cdap-examples/Purchase/src/main/java/co/cask/cdap/examples/purchase/PurchaseApp.java

```diff
@@ -69,7 +70,7 @@ public void configure() {
 
     // Schedule the workflow
     schedule(
-      buildSchedule("DailySchedule", "PurchaseHistoryWorkflow")
+      buildSchedule("DailySchedule", SchedulableProgramType.WORKFLOW, "PurchaseHistoryWorkflow")
```

Comment:
It's just a difference between runtime validation and compile-time validation.
If @chtyim is fine with using ProgramType here, I'll change that.
I think its worth it to introduce a new class so that users automatically know which program types are scheduleable, without reading the schedule documentation.

I'll file a JIRA to cleanup SchedulableProgramType to be more accurate: https://issues.cask.co/browse/CDAP-11646